### ISSUE TITLE
A11Y Lab - Bypass Blocks

### ISSUE THREAD

**Issue by glenn-sorrentino** [scidsg/hushline#162]
Bypass Blocks: A mechanism is available to bypass blocks of content that
are repeated on multiple Web pages.
A keyboard-functional "skip" link should be provided to allow keyboard users to navigate
directly to the main content.
Level Compliance 2.3.1.
A Does not apply
Level Compliance 2.3.2.
AAA Does not apply
Level Compliance 2.3.3.
AAA Does not apply
Level Compliance 2.4.1.
A No
22
The main content of a web page rarely comes first in a typical layout. Before users can
arrive at the main content, they must skip past features like the website's logo and
branding, the site search, the login widget, and the main navigation. In some cases, a
secondary navigation level also appears before the main content. The top section of a
web page can be quite long. Providing a way to skip past all of the features at the top of
a web page helps save keystrokes for keyboard users.
https://hushline.app/
General example:
The Skip Navigation Link allows screen reader users to directly access the main content
of the page without repeatedly going through the navigation on each page.
<div id="skipnav"><a href="#mainContent">Skip navigation</a></div>

<main id="mainContent" tabindex="-1">
 </main>

https://github.com/scidsg/project-info/blob/main/hush-line/1.%20Accessibility/WCAG%202.1%20AAA_HushLine%20website.pdf

**Comment by glenn-sorrentino** [comment]
👍 Fixed

![Screenshot 2023-10-19 at 1 53 26 PM](https://github.com/scidsg/hushline/assets/28545431/26ca8c15-d581-4849-b16e-21abe19a2555)
